description: '[REJECTED]: vfs: check fd has read access in kernel_read_file_from_fd()'
references:
- https://bugzilla.redhat.com/show_bug.cgi?id=2026491
- https://lore.kernel.org/all/20211007220110.600005-1-willy@infradead.org/
- https://lkml.org/lkml/2021/10/6/254
- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-0644
- https://git.kernel.org/linus/032146cda85566abcd1c4884d9d23e4e30a07e9a
- https://www.cve.org/CVERecord?id=CVE-2022-0644
comments:
  debian/carnil: |-
    CVE got rejected by its assigning CNA (Red Hat) because further
    investigation showed that it was not a security issue. Cf.
    https://www.cve.org/CVERecord?id=CVE-2022-0644
  masami: |-
    This bug was introduced by commit b844f0e("vfs: define kernel_copy_file_from_fd()")
    which was merged at 4.6-rc1.
reporters:
- Hao Sun
introduced-by:
  mainline: [b844f0ecbc5626ec26cfc70cb144a4c9b85dc3f2]
fixed-by:
  cip/4.19: [c1ba20965b59c2eeb54a845ca5cab4fc7bcf9735]
  cip/4.19-rt: [c1ba20965b59c2eeb54a845ca5cab4fc7bcf9735]
  cip/5.10: [b721500c979b71a9f02eb84ca384082722c62d4e]
  cip/5.10-rt: [b721500c979b71a9f02eb84ca384082722c62d4e]
  debian/buster-security: ['version:debian/4.19.232-1']
  debian/stretch-security: ['version:debian/4.9.290-1']
  mainline: [032146cda85566abcd1c4884d9d23e4e30a07e9a]
  openela/4.14: [aaa5e83805b09c7ed24c06227321575278e3de1d]
  stable/4.14: [aaa5e83805b09c7ed24c06227321575278e3de1d]
  stable/4.19: [c1ba20965b59c2eeb54a845ca5cab4fc7bcf9735]
  stable/4.9: [52ed5a196b1146e0368e95edc23c38fa1b50825a]
  stable/5.10: [b721500c979b71a9f02eb84ca384082722c62d4e]
  stable/5.4: [0f218ba4c8aac7041cd8b81a5a893b0d121e6316]
  ubuntu/bionic: [3ed7093edabd279f41635416d917f1ec022713b4]
  ubuntu/focal: [6a5bae5268911a7214da4340282db7a00bae4a50]
ignore:
  all: This cve is rejected
